en/components/carousel.md

```diff
@@ -14,7 +14,7 @@ Use the Carousel Component to let the user navigate through a collection of imag
 
```

Comment:
```suggestion

## Indicators and Slides

The Carousel lets you define up to 10 slides, where for each one there is a matching indicator dot, allowing the user to navigate to that respective slide.

```

Reply:
@YoannaIvanova  add an image for this section showing a default slider with 10 dots and just use some image that you like :)